migrations/20190313142702_add_github_updated_at_to_issues.up.fizz

```diff
@@ -0,0 +1,2 @@
+sql("ALTER TABLE public.issues
+ADD COLUMN github_updated_at timestamp without time zone")
```

Comment:
This should probably also be updated for RFC3339

Reply:
I am not well versed with time format handling.
Should I change the query to
`"ALTER TABLE public.issues ADD COLUMN github_updated_at TIMESTAMPTZ"`?
Is this the right way to do it?

Reply:
yes : )

Reply:
Also am I supposed to run `soda migrate up` and also stage the schema.sql in my commit or just the fizz files are supposed to pushed?

Reply:
Yeah stage the schema file as well

> note that you need to rerun the migration you changed in order for the db schema to be upgraded